tests/lib/hardware/node_base.py

```diff
@@ -53,9 +58,23 @@ def get_ssh_ip(self) -> str:
     # disk_attach
     # disk_detach
     @abstractmethod
-    def add_data_disk(self, capacity):
+    def disk_create(self, capacity):
```

Comment:
I think structurally we should have disk_create on the hardware module. This will let us do more advanced tests like moving a disk from one VM to another. It also maps better with the cloud commands that are used. It might need some additional refactoring though so I'm happy for that to be a follow up.

Reply:
That makes sense indeed.
I thought about it but then was not sure this would actually be of any use, because ceph is actually rebalancing data if needed so not sure moving a data drive is actually used.

Let's keep this for another PR so we can move on with OSD related task for now.